### PR TITLE
Create UKDS CKAN Docker Image

### PR DESCRIPTION
**Pull request by chris-daly**

### CODE REVIEW

File: README.md
Comment (CamHurst) [inline README.md:1]: doesnt this need updating to be more specific to our version of CKAN? it reads as though this is a repo for the main CKAN docker image.

**Review by CamHurst — CHANGES_REQUESTED** [review]
code looks fine but readme i think needs updating before approval

File: README.md
Comment (chris-daly) [reply README.md:1]: It is, I've just added a UKDS part to it. It is so, we can merge changes the CKAN team does to their repo. Otherwise future maintenance and updates will be a nightmare.

Comment (CamHurst) [reply README.md:1]: ok thats fine then. ill go ahead and approve

**Review by CamHurst — APPROVED** [review]
changes were not needed to readme, as explained in comment

**Pull request merged**
